vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,10 +472,10 @@ PATCHED_HYPERKUBE_IMAGES="
 1.15.10_f0.0.1
 1.15.10-hotfix.20200326
 1.16.0_v0.0.5
-1.16.7_f0.0.1
```

Comment:
note that for bets perf, this removal requires the week 1's CCP chart bump fully completes before the week 2's RP is rolled out.